_episodes/05-r-telemetry-report-array.md

```diff
@@ -124,7 +127,11 @@ ggsave(plot = proj61_map, filename = "proj61_map.tiff", units="in", width=15, he
 
 ### Mapping my stations - Interactive map
 
-An interactive map can contain more information than a static map.
+An interactive map can contain more information than a static map. Here we will explore the package `plotly` to create interactive "slippy" maps. These allow you to explore your map in different ways by clicking and scrolling through the output.
+
+First, we will set our basemap's aesthetics and bounding box and assign this information (as a list) to a geo_styling variable. Then, we choose which Deployment Metadata dataset we wish to use and identify the columns containing Latitude and Longitude, using the `plot_geo` function. Next, we use the `add_markers` function to write out what information we would like to have displayed when we hover our mouse over a station in our interactive map. In this case, we chose to use `paste` to join together the Station Name and its lat/long. Finally, we add all this information together, along with a title, using the `layout` function, and now we can explore our interactive map!
```

Comment:
My preference would be to break this up and interleave each step with the appropriate code, but that's just a style thing, I leave it up to you to pull the trigger on that if you agree that it's appropriate.